test/statement_test.ts

```diff
@@ -30,9 +30,9 @@ describe('statements', () => {
   });
   it('translates try/catch', () => {
     expectTranslate('try {} catch(e) {} finally {}')
-        .to.equal(' try { } catch ( e ) { } finally { }');
+        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
```

Comment:
Dart, like JS, has `Error`, which has a getter `stackTrace`. We could, instead of `catch(e, s)`, do this:

```dart
} catch(Error e) {
  print(e.stackTrace);
}
```

All Dart's own standard errors are subclasses of `Error` already, and so is our `BaseException`. So it should work just fine.